_pages/en/it-picture-medium-term.md

```diff
@@ -28,7 +28,27 @@ In the following sections, we're trying to take the same end state and create "v
 
 #### Automatic Enrollment and Delivery
 
-- Number of services that require users to submit, or print, PDFs should be reduced by half
+Looking forward, the information required to enroll a Canadian into programs often exists via trusted organizations or other government departments, allowing for the benefits the programs to be made available without the need for a manual process on the behalf of citizens (more details can be found in the [long term plan](https://sara-sabr.github.io/ITStrategy/it-picture-long-term.html)). In order to move in this direction, IITB must move away from manual processes and towards automation and information systems allowing for the secure exchange of information. This aligns well with both the OneGC initiative and the [Policy on Service](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=27916) which reads
```

Comment:
I'd remove the mention of IITB as this is the view of the Citizen.
Instead, recommend rewording the "In order to move in this direction, IITB must...." to
"In order to move in this direction and align to the OneGC intiative (link to GCCollab https://gccollab.ca/groups/profile/154708/enonegc-ungcfrungc), ESDC must move away from manual processes and towards as much automation as possible, leveraging information systems that allows for secure exchange of information with trusted parties. Such information includes private information outside of the program's mandate too should the citizen allow it.

Remove link to the Policy on Service and Digital as this is more internal to ESDC IMHO.
(also the link currently mentioned is the old Policy on Service, the updated one is here https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603)

Reply:
Done (er... doing)